lua/admin.lua

```diff
@@ -75,6 +75,7 @@ drive_on_right = {
 ["Turks and Caicos Islands"] = "false",
 ["Tuvalu"] = "false",
 ["Uganda"] = "false",
+["United Kingdom"] = "false",
```

Comment:
iirc nils removed this because the admin name changed to like great Britain or something or no that each member of the UK became a country level admin poly

Reply:
@kevinkreiser Thank you for catching this issue.  I will merge his updates into mine.

Reply:
https://github.com/valhalla/valhalla/pull/3619